fearless/Common/Operation/EraCountdownOperationFactory/EraCountdownOperationFactory.swift

```diff
@@ -0,0 +1,141 @@
+import RobinHood
+import FearlessUtils
+import SoraKeystore
+
+protocol EraCountdownOperationFactoryProtocol {
+    func fetchCountdownOperationWrapper(targetEra: EraIndex) -> CompoundOperationWrapper<EraCountdown>
+}
+
+enum EraCountdownOperationFactoryError: Error {
+    case noData
+}
+
+final class EraCountdownOperationFactory: EraCountdownOperationFactoryProtocol {
+    let runtimeCodingService: RuntimeCodingServiceProtocol
+    let storageRequestFactory: StorageRequestFactoryProtocol
+    let engine: JSONRPCEngine
+
+    init(
+        runtimeCodingService: RuntimeCodingServiceProtocol,
+        storageRequestFactory: StorageRequestFactoryProtocol,
+        engine: JSONRPCEngine
+    ) {
+        self.runtimeCodingService = runtimeCodingService
+        self.storageRequestFactory = storageRequestFactory
+        self.engine = engine
+    }
+
+    // swiftlint:disable function_body_length
+    func fetchCountdownOperationWrapper(targetEra: EraIndex) -> CompoundOperationWrapper<EraCountdown> {
```

Comment:
If we don't want to calculate time between eras here (we are doing here somewhere in view model factories) then this parameter is useless and error prone. Let's just get rid of it and fetch active era manually

Reply:
done!